.gitignore

```diff
@@ -12,11 +12,12 @@
 .env.backup
 .phpunit.result.cache
 docker-compose.override.yml
-composer.lock
```

Comment:
Why did you remove lock from gitignore?

Reply:
because I don't see the sense of keeping lock in ignor with development .

Reply:
It's a library, not a project. It's useless.